crates/ra_ide_api/src/display/navigation_target.rs

```diff
@@ -51,6 +54,14 @@ impl NavigationTarget {
         self.full_range
     }
 
+    pub fn docs(&self) -> Option<String> {
```

Comment:
A more rustic approach is to avoid clones and return `Option<&str>`

Reply:
Fixed